INSTALL.rst

```diff
@@ -33,43 +33,108 @@ account:
     $ export B2ACCESS_CONSUMER_KEY=...    # the username used for registration
     $ export B2ACCESS_SECRET_KEY=...      # the password used for registration
 
+For demonstration purposes it's recommended to use the staging B2ACCESS
+server. In this case make sure to define and export the
+``USE_STAGING_B2ACCESS`` variable:
 
-1.2 Prerequisite: configuring B2SHARE
+.. code-block:: console
+
+    $ export USE_STAGING_B2ACCESS=1
+
+
+1.2 Prerequisite: B2SHARE secret key
+------------------------------------
+
+The B2SHARE_SECRET_KEY environment variable (a secret string used for
+encrypting user sessions) must also be set as an environment variable. It is
+recommended to use a randomly generated string for this purpose. E.g.:
+
+.. code-block:: console
+
+    $ export B2SHARE_SECRET_KEY=mfTpGpQBwTTOmWCoZrYSuiJ1UzXq1d2L
+
+
+1.3 Prerequisite: configuring B2SHARE
 -------------------------------------
 
+B2SHARE needs connection to external services (e.g. elasticsearch, redis,
+nginx) to provide its features, and uses Docker for simplifying the creation
```

Comment:
we can list all the services here:
redis, elasticsearch, a database (postgresql or mysql), rabbitmq.
Nginx is not required by b2share. We just use it in order to provide https.